gtdynamics/dynamics/Chain.cpp

```diff
@@ -111,20 +140,30 @@ gtsam::Vector3 Chain::DynamicalEquality3(
     // This means that the 3*3 jacobian has an upper triangular structure.
     Matrix A = gtsam::Z_3x3;
 
```

Comment:
Hey Varun,
I understand you moved the adjointMapJacobianQ here, that's fine.
I prefer at this point to leave these jacobian calculations as is.
They are not trivial (at least not for me) and they make sense to me as is right now.
I will eventually get rid of this function.